packages/@okta/vuepress-site/docs/guides/quickstart/using-console/index.md

```diff
@@ -2,39 +2,42 @@
 ---
 
-The Developer Console is the web UI that provides you with a window into your org and lets you configure and manage it. The URL to access your Developer Console consists of your Okta domain plus `-admin`, for example, `https://dev-133337-admin.okta.com`.
+The Admin Console is the web UI that provides you with a window into your org and lets you configure and manage it. The URL to access your Admin Console consists of your Okta domain plus `-admin`, for example, `https://dev-133337-admin.okta.com`.
 
 #### Dashboard
 
-On signing in to Developer Console, you see the Dashboard, a landing page that provides a summary of activity in your org.
+On signing in to Admin Console, you see the Dashboard, a landing page that provides a summary of activity in your org.
 
-You won't yet have much to see if you just created your org, but the Dashboard shows statistics for:
- - Total Users
- - Authentications
- - Failed Logins
+You won't yet have much to see if you just created your org, but the Dashboard shows various statistics group into several different widgets. These widgets include:
+
+- **Overview:**: Shows totals and other statistics for your org's users, groups and apps.
+- **Status:**: Lists the operation status of the Okta service and any running agents.
+- **Tasks:**: Lists issues in your org that require attention. You can filter by active (default) or unsubscribed tasks.
+- **Org changes:**: Displays system logs showing all the org changes in the last 7 days.
+- **Security Monitoring:**: Displays security related information such as recommended security tasks and suspicious user activity.
 
 A search bar at the top of the Dashboard lets you quickly bring up information on a specific user or app.
 
 #### System Log
 
-The latest events logged in your org's System Log are displayed in the lower part of the dashboard. The System Log records actions taken by you and any other admins, as well as ongoing events that occur for each user, like user creation and authentication attempts. You can click each event to expand it and get full details.
+Some of the statistics on the Dashboard are derived from the org's System logs. You can access the System logs page by either clicking on the  **View all** link in the **Org Changes** widget or the suspicious activity **View** link in the **Security Monitoring** widget. The **System Logs** can also be access by selecting **Reports > System Logs** in the side navigation. The System Log records actions taken by you and any other admins, as well as ongoing events that occur for each user, like user creation and authentication attempts. You can click each event to expand it and get full details.
 
-In the upper-right corner of the screen, you can see the Okta domain of the org you’re in. This can be handy if you manage multiple orgs.
+In the global header on the the upper-right corner of the screen, you can view the Okta domain of the org you’re in. This can be handy if you manage multiple orgs.
```

Comment:
```suggestion
In the global header in the upper-right corner of the page, you can view the Okta domain of the org that you’re in. This can be handy if you manage multiple orgs.
```